pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -155,6 +157,9 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
 
     tests = {
       inherit (nixosTests) tandoor-recipes;
+      runTandoorRecipes = runCommand "run-${pname}" {} ''
+        DEBUG_TOOLBAR=0 ${tandoor-recipes}/bin/tandoor-recipes > $out
+      '';
```

Comment:
I don't see what this gets us compared to the NixOS test which is already included. Could you explain the need?

Reply:
It's useful to show end users that the original command comes broken out of the box, and that they need to set this variable to be able to run it.

Reply:
The NixOS module already mentions that users should use `/var/lib/tandoor-recipes/tandoor-recipes-manage`, which is the supported way of using the manage script.

If you find the documentation lacking, I would suggest expanding on it, rather than add a redundant test which can only be seen by looking at the package's source.